### DISCUSSION TITLE
How to generate DTS while packaging

### DISCUSSION THREAD

**Discussion by AhMisty** [rolldown/rolldown#2996] [Q&A] [answered]
How to generate DTS while packaging

**Comment by 7086cmd** [comment, marked as answer]
You may use [`unplugin-isolated-decl`](https://github.com/unplugin/unplugin-isolated-decl) as a plugin for DTS generation currently.

**Comment by AhMisty** [comment]
ok

**Comment by braebo** [comment]
I saw support for this in the [roadmap](https://github.com/rolldown/rolldown/discussions/153) under __Side Features__ and was hoping to subscribe to a GitHub issue for the feature so I know when it's safe to remove `unplugin-isolated-decl` :)
